lib/ansible/modules/cloud/openstack/os_group.py

```diff
@@ -45,6 +45,11 @@
         - Group description
      required: false
      default: None
+   domain_id:
+     description:
+        - Domain id to create the group in if the cloud supports domains.
+     required: false
+     default: None
```

Comment:
please add the  version added:

version_added: "2.3"

Something like https://github.com/ansible/ansible/blob/devel/lib/ansible/modules/cloud/openstack/os_subnet.py#L77 as an example.

Other than that, change lgtm, I will merge when amended.